AsyncDisplayKit/ASImageNode+AnimatedImage.h

```diff
@@ -0,0 +1,23 @@
+//
+//  ASImageNode+AnimatedImage.h
+//  Pods
+//
+//  Created by Garrett Moon on 3/22/16.
+//
+//
+
+#import "ASImageNode.h"
+
+#import "ASImageProtocols.h"
+
+@interface ASImageNode ()
```

Comment:
Is it needed to have two different categories?  Nit: remove space between imports and above/below properties, IMO the more distinct groupings are easier to read (might just be tired)